src/libs/kata-types/src/config/hypervisor/mod.rs

```diff
@@ -379,6 +379,17 @@ pub struct DebugInfo {
     #[serde(default)]
     pub enable_debug: bool,
 
+    /// The log log level will be applied to hypervisor.
+    /// Possible values are:
+    /// - trace
+    /// - debug
+    /// - info
+    /// - warn
+    /// - error
+    /// - critical
+    #[serde(default)]
+    pub log_level: String,
```

Comment:
I am not sure if we should set ‘real default’ for the value, like 'info'. Seems now the default value is empty